### DISCUSSION TITLE
Implementation of variable mint fee

### DISCUSSION THREAD

**Discussion by ghost** [tracer-protocol/perpetual-pools-contracts#282] [General]
[Technical Specification](https://www.notion.so/Variable-Mint-Fee-Technical-Specifications-9ed931c9494b43eb9ceafaa2c49582e2)
[Background document for reference](https://www.notion.so/V2-Minting-and-Burning-Fee-4237b7b4d5104991976b8be7f1129bf9)

**Comment by ghost** [comment]
@greenbookwebb @issy-fleming follow up question from @mynameuhh and @Paul4912 is "What changes need to made to the burn fee, if any"?
Do we need to remove burn fees entirely or are they to be set to a flat fee on deployment?

**Reply by CalabashSquash** [reply]
>  Do we need to remove burn fees entirely or are they to be set to a flat fee on deployment?

Another possibility is that they have their own function for modifying itself.

**Reply by greenbookwebb** [reply]
We will just be setting the burn fee to zero for time being. I assume this would requre no change in the mechanism, and just a setter call.

**Reply by CalabashSquash** [reply]
> We will just be setting the burn fee to zero for time being. I assume this would requre no change in the mechanism, and just a setter call.

Is it possible the burn fee will change in the future?
Is it possible the burn fee will also have a self-updating mechanism, like what is being proposed with the mint fee, at some point in the future?

**Reply by greenbookwebb** [reply]
I don't check github, btw, and don't get notifications to this. 

How do y'all do it? Do y'all just check in every now and then, or do y'all have push notifications?

**Reply by CalabashSquash** [reply]
> How do y'all do it? Do y'all just check in every now and then, or do y'all have push notifications?

Yeah I raised this with rogue fenix while he made this github discussion, because I knew you were not likely to see it but I'm not sure what his reasoning was.

**Reply by greenbookwebb** [reply]
The burn fee parameter may change in future (but it's very unlikely). 

We will not be doing any complex self-updating mechanism for burn fee like we did for mint fee. It will just be fixed. @mynameuhh

**Reply by CalabashSquash** [reply]
Yep makes sense. TY

**Reply by CalabashSquash** [reply]
@greenbookwebb Do you want the `mintFee` itself to be modifiable by governance, or just the `Change_Interval`?

**Reply by greenbookwebb** [reply]
@mynameuhh both please
